$OpenBSD: patch-configure,v 1.10 2012/08/21 17:27:13 ajacoutot Exp $

Disable --as-needed to prevent undefined symbol on pthread_*

Fix DATADIRNAME: see LOCALBASE/share/aclocal/glib-gettext.m4.

--- configure.orig	Wed Aug  8 13:59:34 2012
+++ configure	Tue Aug 21 16:56:37 2012
@@ -13642,10 +13642,6 @@ else
 fi
 
 
-if test "x$GCC" = "xyes"; then
-  LDFLAGS="-Wl,--as-needed $LDFLAGS"
-fi
-
 # i18n support
 
   { $as_echo "$as_me:${as_lineno-$LINENO}: checking whether NLS is requested" >&5
@@ -14191,6 +14187,9 @@ else
   DATADIRNAME=lib
 fi
 
+    ;;
+    *-*-openbsd*)
+    DATADIRNAME=share
     ;;
     *)
     DATADIRNAME=lib
